quarkus/dist/src/main/content/bin/kc.sh

```diff
@@ -1,5 +1,32 @@
 #!/bin/bash
 
+# With added utility function.
+function printArrayElementsInLines() {
+    local sourceArray=${!1}
+    printf "%s\n" ${sourceArray[@]}
+}
+
+function checkIfArrayContains() {
```

Comment:
I appreciate the attempt to improve this bash script, although we are actually already doing "contains" in a different way in the same file:
https://github.com/keycloak/keycloak/blob/e28b5cf15b3d42daf055b19a8508ee3284f07a98/quarkus/dist/src/main/content/bin/kc.sh#L139-L140

Reply:
Ok. Change done. 

Thanks for this review and suggestion.